cmd/plugin/cli/install.go

```diff
@@ -50,22 +50,24 @@ func InstallCmd() *cobra.Command {
 	cmd.Flags().String(installer.StosVersionFlag, "", "version of storageos operator")
 	cmd.Flags().String(installer.StosOperatorYamlFlag, "", "storageos-operator.yaml path or url")
 	cmd.Flags().String(installer.StosClusterYamlFlag, "", "storageos-cluster.yaml path or url")
+	cmd.Flags().String(installer.StosPortalConfigYamlFlag, "", "storageos-portal-configmap.yaml path or url")
```

Comment:
`storageos-portal-manager-configmap.yaml` is a bit longer, but we use this form at other places. WDYT?